.travis.yml

```diff
@@ -27,6 +27,7 @@ install:
   - git clone --depth=1 https://github.com/OCA/maintainer-quality-tools.git ${HOME}/maintainer-quality-tools
   - export PATH=${HOME}/maintainer-quality-tools/travis:${PATH}
   - travis_install_nightly
+  - printf '[options]\n\nrunning_env = dev\n' > ${HOME}/.openerp_serverrc
```

Comment:
Do we still need thaht ?, .openerp_serverrc seems pretty old for me

Reply:
I have added this change at ff29eeb to fix errors reported by Travis at https://travis-ci.org/OCA/server-tools/builds/569737181?utm_source=github_status&utm_medium=notification
```
  File "/home/travis/dependencies/server-env/server_environment/serv_config.py", line 50, in <module>
1599    "The parameter 'running_env' has not be set neither in base config "
1600Exception: The parameter 'running_env' has not be set neither in base config file option -c or in openerprc.
1601We strongly recommend against using the rc file but instead use an explicit config file with this content:
1602[options]
1603running_env = dev
```
I don't remember exactly where but I found this ``.travis.yml`` line in one of C2C's (not that old?) projects and it seems to do the job :)

Reply:
I have tried to use ``.odoo_serverrc`` (but not sure it really exists, I have found no occurence of it) or ``.odoorc``. But with no success.